buffer/src/main/java/io/netty/buffer/PoolArenaMetric.java

```diff
@@ -120,6 +125,11 @@ public interface PoolArenaMetric extends SizeClassesMetric {
      */
     long numHugeDeallocations();
 
+    /**
+     * Return the number of chunk deallocations done via the arena.
+     */
+    long numChunkDeallocations();
```

Comment:
Adding methods to interfaces is not possible as it's a breaking change. We could do it in the 4.2 branch with default methods tho

Reply:
Thanks, I have reopened in #14262